Makefile

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
Is the plan to do cmake stuff later?

Reply:
hmm ok there's a cmake file later....

Reply:
I'm using `make` to drive my builds even if the main work is done by `cmake`: the default target just runs `cmake` to build and test things. There are a few other target like a naïve dependency checker and something to build one header suitable for Compiler Explorer: I'd forget how to run these if I don't encode the "procedure" somewhere.

tl;dr: it uses `cmake` already! `make` is just a "convenience" frontend you don't have to use.